draft-ietf-lake-authz.md

```diff
@@ -892,16 +892,18 @@ If the authentication credential CRED_V cannot be used in a TLS handshake, e.g.,
 V MUST then perform an EDHOC session over the TLS connection proving to W the possession of the private key corresponding to CRED_V.
 Performing the EDHOC session is only necessary if V did not authenticate with CRED_V in the TLS handshake with W.
 
-Editor's note: Clarify that performing TLS handshake is not necessary for each device request; if there already is a TLS connection between V and W that should be reused. Similar considerations for 5.2 and 5.3.
-
+Since the relationship between V and W is long-lived, it is RECOMMENDED to establish the TLS connection once and reuse it for multiple VREQ/VRES exchanges.
+While a policy for renewal of the TLS connection should be applied, it is out of scope of this document.
```

Comment:
```suggestion
The relationship between V and W is long-lived.  HTTP/1.1 and higher support persistent connections, and SHOULD be used in order to reduce overhead if a flood of new devices need to be onboarded.  Support for TLS session resumptions tickets {{RFC8446, Section 2.2}} is appropriate for longer term associations.
While a policy for renewal of the TLS connection should be applied, it is out of scope of this document.
```

{Do we really need to say anything?  This connection between non-constrained systems on non-challenged networks should not be controversial}